Makefile

```diff
@@ -49,8 +49,8 @@ all: generate fix vet fmt manifests test lint license misspell tidy bin/kube-app
 ## Tooling Binaries
 ## --------------------------------------
 
-$(TOOLBIN)/controller-gen:
-	GOBIN=$(TOOLBIN) GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.5
+$(TOOLBIN)/controller-gen: $(TOOLBIN)/kubectl
+	GOBIN=$(TOOLBIN) GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0
```

Comment:
Should this be updated to v0.4.0? When I originally created this commit, v0.4.0 was not out.

Reply:
controller-gen@v0.4.0 seems to be incompatible with client-go@v0.18.2. I got the following error when updating to v0.4.0:
```
17:14 $ make manifests 
/usr/local/google/home/nanyu/go/src/sigs.k8s.io/application/hack/tools/bin/controller-gen \
        "crd:trivialVersions=true,crdVersions=v1" \
        rbac:roleName=kube-app-manager-role \
        paths=./... \
        output:crd:artifacts:config=config/crd/bases \
        output:crd:dir=config/crd/bases \
        output:webhook:dir=config/webhook \
        webhook
../../../../pkg/mod/k8s.io/client-go@v0.18.2/discovery/discovery_client.go:30:2: module github.com/googleapis/gnostic@latest found (v0.5.1), but does not contain package github.com/googleapis/gnostic/OpenAPIv2
../../../../pkg/mod/k8s.io/kube-openapi@v0.0.0-20200805222855-6aeccd4b50c6/pkg/util/proto/document.go:24:2: case-insensitive import collision: "github.com/googleapis/gnostic/openapiv2" and "github.com/googleapis/gnostic/OpenAPIv2"
Error: not all generators ran successfully
run `controller-gen crd:trivialVersions=true,crdVersions=v1 rbac:roleName=kube-app-manager-role paths=./... output:crd:artifacts:config=config/crd/bases output:crd:dir=config/crd/bases output:webhook:dir=config/webhook webhook -w` to see all available markers, or `controller-gen crd:trivialVersions=true,crdVersions=v1 rbac:roleName=kube-app-manager-role paths=./... output:crd:artifacts:config=config/crd/bases output:crd:dir=config/crd/bases output:webhook:dir=config/webhook webhook -h` for usage
```

I would keep using v0.3.0 with K8s v1.18.2

Reply:
I refreshed all the dependencies and it worked now with controller-gen `v0.4.0`.